README.md

```diff
@@ -1,3 +1,9 @@
+This is a fork of [word2gauss](https://github.com/seomoz/word2gauss) that has been modified to run on Python 3 (I attempted to preserve compatibility with Python 2.7, but I have only tested the code on Python 3.5, and Python 2 compatibility may have suffered).
```

Comment:
Assuming we accept this merge, we wouldn't want this change to README.md merged. There would just be a single version of word2gauss which supported both Python 2 and 3.